webpack.config.js

```diff
@@ -57,6 +32,35 @@ module.exports = {
         exclude: /node_modules/,
         loader: 'babel-loader',
       },
+      {
+        test: /\.css$/,
+        use: [ 'style-loader', 'css-loader' ]
+      }
     ],
   },
 };
+
+if (env === 'production') {
+  baseConfig.plugins.push(new OfflinePlugin());
+  baseConfig.optimization = {
```

Comment:
I think this is automatically ignored in development.

Reply:
To show a different theme in the theme playground